src/tools/wasm-reduce.cpp

```diff
@@ -321,6 +321,7 @@ struct Reducer : public WalkerPass<PostWalker<Reducer, UnifiedExpressionVisitor<
     Module wasm;
     ModuleReader reader;
     reader.read(working, *module);
+    wasm.features = FeatureSet::All;
```

Comment:
I'm not sure this is optimal - it might lead the optimizer to attempt odd things that hurt reduction. Please add a comment that we should likely autodetect features from the binary here instead, once we can do that.